src/content/posts/how-to-raise-funds/index.md

```diff
@@ -0,0 +1,118 @@
+- **Start Date and Expiry Date**: Set the start date and the expiry date for your project.
+- **Penalty Days**: Define the number of penalty days.
+- **Target Amount**: Specify the total amount of funding you aim to raise.
+- **Define Project Stages**:
+  - Allocate a percentage of the total funds to each stage.
+  - Set the date for each stage.
+  - Click "Next" to proceed to the next step.
+
+    ![Project Info](./assets/project-info.svg)
+
+### Step 3: On-Chain Confirmation
+
+- **Review Project Details**: Review all the details of your project.
+- **Confirm Project Creation**: Once all the information is verified, click on "Submit" to finalize the creation of your project on the blockchain.
+
+    ![On-Chain Confirmation](./assets/on-chain.svg)
+
+
+### Proceeding to the Next Section
+After filling out all the required information in the Project Info section, click the "Next" button to move on to the On Chain section, where you will define blockchain-related parameters for your project.
+
+![Example Project View](./assets/view-project.svg)
+
+### Post Project Updates on Nostr
+
+- Export the private key from Angor.
+- Import the private key into a Nostr client.
+- Post updates on project progress and milestone completion on Nostr.
+- Ensure updates are clear and informative for investors.
+
+### Approving Investment Requests
+- Once your project is live on Angor, you may start receiving investment requests from potential investors.
+- Review Requests: Navigate to the "Actions" section in your project dashboard to view all pending signatures.
+
+  ![Approve signature tab](./assets/approve-signature-tab.svg)
+
+- Approve Signatures: Approve the request through Angor. This action confirms acceptance of the investor’s contribution to your project.
+
+  ![Approving investment by founder](./assets/approving-investment.svg)
+
+### Spend Funds for Milestones
+
+- As a founder, once a milestone is reached, sign the transaction to spend the funds for that milestone.
+- Ensure the spending aligns with the milestone requirements and project goals.
+
+  ![Founder stage funds claim](./assets/founder-stage-claim.svg)
+
+### Claiming Funds 
+Easily monitor project progress, release milestone funds, and handle penalties directly from your Angor dashboard.
+
+#### 1. Monitor Project Progress
+
+- Regularly check the project updates on Angor.
+- Ensure that the milestones are being met as planned.
+
+#### 2. Initiate Fund Release for Milestones
+
+- Once a milestone is reached, navigate to your project dashboard.
+- Find the milestone that has been achieved and click on the "Claim Funds" button.
+- Sign the transaction to release the funds for that milestone.
+
+#### 3. Regular Fund Claim Process:
+
+- Navigate to the "Funds" section in your project dashboard.
+- Click on the "Claim Funds" button for the available milestone funds.
+- Sign the transaction to transfer the funds to your wallet.
+  
+  ![Portfolio Image](./assets/invested-portfolio.svg)
```

Comment:
this is the image for the investor not the founder so no need to show it, instead you can show the project page with the progress bar of investor